common/thorhelper/thorsoapcall.cpp

```diff
@@ -1182,6 +1203,7 @@ class CWSCHelper : implements IWSCHelper, public CInterface
 
         if (wscMode == SCrow)
         {
+            activitySpanScope->setSpanAttribute("activity.mode", "SCrow");
```

Comment:
I don't think activity.mode will be worth recording.

Reply:
seems reasonable, I'll remove